UKHomeOffice/nmsw-ui
# Fix failing test

- Fix failing test with css locator change 
- Update failing fixture files
- Fix erroring for FAL5 template upload


## To test

Run with cypress runner
```
npm run cypress:open:dev -- --env MAIL_API_KEY=${MAIL_API_KEY}
```
and choose passenger-status-check.feature
fal5-error-validation.feature
or in cmd line
```
npm run cypress:run:dev -- --env MAIL_API_KEY=${MAIL_API_KEY} --spec cypress/e2e/features/passenger-status-check.feature

```

----

## Notes
